main.go

```diff
@@ -661,6 +667,16 @@ func run(state overseer.State) {
 		if err := e.ScanElasticsearch(ctx, cfg); err != nil {
 			logFatal(err, "Failed to scan Elasticsearch.")
 		}
+	case jenkinsScan.FullCommand():
+		cfg := engine.JenkinsConfig{
+			Endpoint:              *jenkinsEndpoint,
```

Comment:
I think `jenkinsEndpoint` should be renamed to `jenkinsURL` to match the CLI flag arg name.